streamly.cabal

```diff
@@ -388,7 +388,7 @@ library
                        Streamly.Prelude
                      , Streamly.Data.Unfold
                      , Streamly.Data.Fold
-                     , Streamly.Data.Array.Storable.Foreign
```

Comment:
You should deprecate this and update the changelog

Reply:
It was never released as Data.Array.Storable.Foreign AFAICT. Memory.Array's deprecation notice is appropriately updated.

Reply:
Yeah, you're right. My bad.